### PR TITLE
CORE-4485 Thorium custom error changes

### PR DESCRIPTION
**Pull request by chriswong929**

### CODE REVIEW

**Comment by Buktoria** [comment]
Update thorium version. Also create an flux pr for the version update. Update tests

**Comment by chriswong929** [comment]
Updated tests. Bumped version number

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='Thorium',
-    version='0.2.13',
+    version='0.3.0',
```
Comment (Buktoria) [inline setup.py]: change to 0.2.14

Comment (chriswong929) [inline setup.py]: Same thing here https://github.com/EventMobi/flux-client-python/pull/34#discussion_r62243070

**Comment by rjeaster** [comment]
reviewed

**Comment by chriswong929** [comment]
Changed version from `0.3.0` to `0.2.14`

**Pull request merged**